src/lib/snack-bar/simple-snack-bar.scss

```diff
@@ -11,14 +11,19 @@ $mat-snack-bar-button-margin: 48px !default;
 }
 
 .mat-simple-snackbar-action {
-  @include mat-button-reset;
-
-  background: none;
+  display: flex;
+  flex-direction: column;
   flex-shrink: 0;
-  margin-left: $mat-snack-bar-button-margin;
+  justify-content: space-around;
+  margin: -8px 0 -8px $mat-snack-bar-button-margin;
+
+  button {
+    flex: 1;
+    max-height: 36px;
```

Comment:
Move this into a variable or add a comment about where it's coming from?

Reply:
Done